superset-frontend/plugins/plugin-chart-table/src/utils/DateWithFormatter.ts

```diff
@@ -31,19 +33,12 @@ export default class DateWithFormatter extends Date {
 
   constructor(
     input: DataRecordValue,
-    {
-      formatter = String,
-      forceUTC = true,
```

Comment:
here we remove the `forceUTC` parameter, as it's never used, and likely never will be, as Superset assumes that timestamps should always be naive (any timezone logic needs to happen in the backend).